src/Symfony/Component/DependencyInjection/Tests/Compiler/AutowirePassTest.php

```diff
@@ -81,38 +84,36 @@ public function testProcessVariadic()
 
     public function testProcessAutowireParent()
     {
-        $this->expectException('Symfony\Component\DependencyInjection\Exception\RuntimeException');
-        $this->expectExceptionMessage('Cannot autowire service "c": argument "$a" of method "Symfony\Component\DependencyInjection\Tests\Compiler\C::__construct()" references class "Symfony\Component\DependencyInjection\Tests\Compiler\A" but no such service exists. You should maybe alias this class to the existing "Symfony\Component\DependencyInjection\Tests\Compiler\B" service.');
         $container = new ContainerBuilder();
 
         $container->register(B::class);
         $cDefinition = $container->register('c', __NAMESPACE__.'\C');
         $cDefinition->setAutowired(true);
 
         (new ResolveClassPass())->process($container);
-        (new AutowirePass())->process($container);
-
-        $this->assertCount(1, $container->getDefinition('c')->getArguments());
```

Comment:
`container->getDefinition('c')->getArguments()` equals to 0.
This assert were never reached because call to `process` triggers an exception and cautch bad the former `$this->expectException('Symfony\Component\DependencyInjection\Exception\RuntimeException');`